server/src/gateways/judges.gateway.ts

```diff
@@ -53,13 +56,24 @@ export class JudgesGateway implements FightEndConditionFulfilledObserver {
     }
 
     const fight: FightImpl = this.fightsService.getFight(fightId);
+    const role = fight.getJudgeRole(judgeId);
+    const connectedJudges = fight.getConnectedJudgesStatus();
+    connectedJudges.delete(role);
+
     const response: JoinResponse = {
+      connected: [...connectedJudges.keys()],
       status: status,
       role: fight.getJudgeRole(judgeId),
       redPlayer: await this.playersService.getPlayer(fight.redPlayer.id),
       bluePlayer: await this.playersService.getPlayer(fight.bluePlayer.id),
     };
 
+    for (const [judgeType, judgeState] of connectedJudges.entries()) {
+      judgeState.socket.emit(JudgesSocketEvents.JudgeJoined, {
+        newJudge: role,
+      });
+    }
+
```

Comment:
Przesyłanie tej informacji można załatwić w jednej linijce korzystając z zaimplementowanej wyżej funkcji `sendToAllJudges(fight, JudgesSocketEvents.JudgeJoined, {newJudge: role})`

Reply:
send to all judges wyśle update też do sędziego, który wlasnie dołączył - nw czy warto

Reply:
Wydaje mi się, że to w niczym nie przeszkadza. Ale masz rację, że to niekoniecznie jest pożądane zachowanie - w takim razie możesz zmienić albo zostawić jak jest, wedle własnego uznania.